src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -37,7 +37,7 @@ import { getConditionalPages } from '../../utilities';
 import SupportingDocsVerification from './supportingDocsVerification';
 import MissingFileList from './MissingFileList';
 
-const mailInfo = (showOpt = true, address, officeName, faxNum) => {
+const mailInfo = (showOpt = true, address, officeName, faxNum, preamble) => {
```

Comment:
We now accept the `preamble` argument which can have JSX in it. If it's not present we default to the original copy.